### ISSUE TITLE
The fuzzy_like_this or flt query has been removed. Instead use the fuzziness parameter with the match query or the More Like This Query.

### ISSUE THREAD

**Issue by fancyecommerce** [yiisoft/yii2-elasticsearch#169]
yii2-elasticsearch   : https://github.com/yiisoft/yii2-elasticsearch/blob/master/docs/guide/usage-ar.md

```
// http://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-flt-query.html
$query = Article::find()->query([
    "fuzzy_like_this" => [
        "fields" => ["title", "description"],
        "like_text" => "This query will return articles that are similar to this text :-)",
        "max_query_terms" => 12
    ]
]);
```

but 👍 

https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-flt-query.html

```
Fuzzy Like This Queryedit
The fuzzy_like_this or flt query has been removed. Instead use the fuzziness parameter with the match query or the More Like This Query.
```

**Comment by sokolnikov911** [comment]
Fixed in PR #170

**Comment by beowulfenator** [comment]
PR already merged.
